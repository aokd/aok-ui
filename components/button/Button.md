#### 代码演示

    <Button/>
