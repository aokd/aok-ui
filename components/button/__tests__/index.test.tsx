import { render } from 'enzyme'
import React from 'react'
import Button from '..'

describe('Button', () => {
  it('renders correctly', () => {
    const wrapper = render(<Button>Demo</Button>)
    expect(wrapper).toMatchSnapshot()
  })
})
