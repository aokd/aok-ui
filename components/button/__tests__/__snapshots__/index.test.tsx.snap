// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Button renders correctly 1`] = `
<button
  class="aok-btn aok-btn-default"
  type="button"
>
  Demo
</button>
`;
